src/Http/Controllers/Admin/XlsformTemplateCrudController.php

```diff
@@ -89,9 +100,14 @@ protected function setupCreateOperation(): void
             ->type('textarea')
             ->validationRules('nullable');
 
+        CRUD::field('xlsformSubject')
+            ->type('relationship')
+            ->label('Xlsform Subject')
+            ->validationRules('required');
+
```

Comment:
Again, I think we need some more information for the user for them to correctly select the "Xlsform Subject".